cmd/nex/workload.go

```diff
@@ -313,11 +321,34 @@ func (i InfoWorkload) Run(ctx context.Context, globals *Globals) error {
 		return err
 	}
 
+	if i.Json {
+		out, err := json.Marshal(resp)
+		if err != nil {
+			return err
+		}
+		fmt.Println(string(out))
+		return nil
+	}
+
 	w := columns.New("Information about Workload %s", i.WorkloadId)
 	w.AddRow("Name", resp.WorkloadSummary.Name)
 	w.AddRow("Start Time", resp.WorkloadSummary.StartTime)
-	w.AddRowIf("Run Time", resp.WorkloadSummary.Runtime, resp.WorkloadSummary.Runtime != "")
+	w.AddRow("Run Time", func() string {
+		if resp.WorkloadSummary.Runtime == "" {
```

Comment:
Once upon a time, this as called execution time, and the consensus was that was too aggressive of a term.  The problem with an EndTime, is that will never be non-0 because workloads that have ended will not show here.  I agree with "Runtime" being confusing.  Is there another word we can use, or can "execution time" be reconsidered

Reply:
> The problem with an EndTime, is that will never be non-0 because workloads that have ended will not show here

Ack.. as an aside, it would be interesting to think about how we might want to store metadata on completed workloads.

Since run/exec time only applies to running workloads, wouldn't the total time be calculated relative to _now_ when this is reported?

Reply:
chatted sync